pydis_site/apps/content/resources/guides/pydis-guides/contributing.md

```diff
@@ -91,26 +79,55 @@ Our projects on Python Discord are open source and [available on Github](https:/
   </div>
 </div>
 
-If you don't understand anything or need clarification, feel free to ask any staff member with the  **@PyDis Core Developers** role in the server. We're always happy to help!
+# How do I start contributing?
+  Unsure of what contributing to open source projects involves? Have questions about how to use GitHub? Just need to know about our contribution etiquette? Completing these steps will have you ready to make your first contribution no matter your starting point.
+
+  Feel free to skip any steps you're already familiar with, but please make sure not to miss the  [Contributing Guidelines](#5-read-our-contributing-guidelines).
+
+  If you are here looking for the answer to a specific question, check out the sub-articles in the top right of the page to see a list of our guides.
+
+  **Note:** We use Git to keep track of changes to the files in our projects. Git allows you to make changes to your local code and then distribute those changes to the other people working on the project. You'll use Git in a couple steps of the contributing process. You can refer to this [**guide on using Git**](./working-with-git/).
+  {: .notification }
+
+### 1. Fork and clone the repo
+  GitHub is a website based on Git that stores project files in the cloud. We use GitHub as a central place for sending changes, reviewing others' changes, and communicating with each other. You'll need to fork the repository to create a copy under your own GitHub account. You'll make your changes to this copy, which can then later be merged into the PyDis repository.
+
+  Check out our [**guide on forking a GitHub repo**](./forking-repository/).
+
+  Now that you have your own fork you need to be able to make changes to the code. You can clone the repo to your local machine, commit changes to it there, then push those changes to GitHub.
+
+  Check out our [**guide on cloning a GitHub repo**](./cloning-repository/).
+
+### 2. Set up the project
+  You have the source code on your local computer, now how do you actually run it? We have detailed guides on setting up the environment for each of our main projects:
+
+  * [**Sir Lancebot**](./sir-lancebot/)
+
+  * [**Python Bot**](./bot/)
+
+  * [**Site**](./site/)
+
+### 3. Read our Contributing Guidelines
+  We have a few short rules that all contributors must follow. Make sure you read and follow them while working on our projects.
+
+  [**Contributing Guidelines**](./contributing-guidelines/).
+
+  As mentioned in the Contributing Guidelines, we have a simple style guide for our projects based on PEP 8. Give it a read to keep your code consistent with the rest of the codebase.
```

Comment:
Why is there indentation?

Reply:
There isn't anymore

Reply:
I am marking this unresolved because there is still for "How do I start contributing?".

Reply:
There isn't anymore 2